### ISSUE TITLE
Use correct service name on Debian-based systems

### ISSUE THREAD

**Issue by natefoo** [galaxyproject/ansible-postgresql#23]
Recent Debian systems get a service named `postgresql.service` but restarting this actually does not restart PostgreSQL, for this you have to restart `postgresql@<version>-<cluster>` (e.g. `postgresql@10-main`).

**Comment by MichaelDBA** [comment]
This is a major problem that you cannot use **systemd** to start the cluster installed by this ansible role.  It has not been addressed in almost 2 years.  Would you please address why this has not happened and what is the roadmap for this ansible role in the future. i.e, is it going to be maintained?

**Comment by hexylena** [comment]
>  It has not been addressed in almost 2 years

yes we've been having a global pandemic, our bandwidth is lower than normal. galaxyproject is specifically a bioinformatics project heavily used during covid times, so, you can understand.

> is it going to be maintained

we will be continuing to use it in production in a number of sites. It is up to you if you find this role useful or wish to find another one, or fork this one.

**Comment by MichaelDBA** [comment]
Whoops, I accidentally deleted my issue comments.  Thanks for your responses.  Regarding the last comment, are you going to fix the restart handler so a restart can be done via ansible?

**Comment by hexylena** [comment]
Yes, it's very likely we will fix this, especially now that ubuntu support will grow as centos is ended. I cannot provide a timeline I'm afraid.
